README.md

```diff
@@ -65,20 +65,66 @@ yarn audit
 
 ### Configuration
 
-You don't need to [Configure Gimbal](./packages/gimbal/docs/config), but we understand that defaults are _optimistic_, at least for existing projects that want to introduce performance budgeting. 
+You don't need to [Configure Gimbal](./packages/gimbal/docs/config), but we understand that defaults are _optimistic_, at least for existing projects that want to introduce performance budgeting.
 
-Here's a **sample `.gimbalrc.yml` config** file:
+To ease you be ready to use, let's start with some **sample `.gimbalrc.yml` config** files:
+
+1. Minimal
+2. Minimal with all native audits
+3. Using other audit plugins and more sample configurations
+
+You may save them as your `.gimbalrc.yml` file and run `gimbal`.
+
+#### 1) Minimal sample `.gimbalrc.yml` config file:
+
+```yml
+audits:
+  - size
+
+configs:
+  buildDir: build
+```
+
+#### 2) Minimal sample `.gimbalrc.yml` config file running all native audits:
 
 ```yml
+audits:
+  - size
+  - lighthouse
+  - heap-snapshot
+  - unused-source
+
 configs:
-  # Specify audits to run. Also include any plugins (like axe)
-  audits:
+  buildDir: build
+```
+
+#### 3) Sample `.gimbalrc.yml` config file running audit plugins and more configurations:
+
+Before executing this config file you should install mentioned plugins. For instance:
+
+```sh
+# with npm
+npm install @modus/gimbal-plugin-axe @modus/gimbal-plugin-last-value @modus/gimbal-plugin-sqlite
+
+# or with yarn
+yarn add @modus/gimbal-plugin-axe @modus/gimbal-plugin-last-value @modus/gimbal-plugin-sqlite
+```
+
+In case you don't use `build` as your build directory and an exception raises with an error 
+concerning a nonexistent `build` directory, please create one and run gimbal again.
```

Comment:
I am not sure yet which configuration requires this. But we have an exception when `build` directory does not exist.

Reply:
We need to document all config properties someplace and note their default values. 
Also, it's worth mentioning that users need to build the project before running gimbal. That's where all the juice is

Reply:
Sure. I will mention that users need to build.